src/components/calcite-value-list-item/calcite-value-list-item.scss

```diff
@@ -1,7 +1,12 @@
 :host {
-  @extend %component-host;
   @apply mb-px flex shadow-border-bottom;
   transition: background-color $transition, box-shadow $transition;
+  * {
+    @apply box-border;
+  }
+  background-color: var(--calcite-ui-foreground-1);
+  color: var(--calcite-ui-text-2);
+  font-size: var(--calcite-font-size--1);
```

Comment:
Use `@apply` here.

Reply:
Also noticed that `.handle` has a text-color rule. We probably don't need to do that.